lib/membrane_h264_plugin/parser/nalu_parser.ex

```diff
@@ -27,86 +25,68 @@ defmodule Membrane.H264.Parser.NALuParser do
   def new(),
     do: %__MODULE__{
       scheme_parser_state: SchemeParser.new(),
-      has_seen_keyframe?: false,
-      prev_pts: nil,
-      prev_dts: nil
+      has_seen_keyframe?: false
     }
 
   @doc """
   Parses a `Membrane.Buffer.t()` with a payload being a part of H264 stream.
 
   As a result it returns a list of NAL units, along with a part of a payload
-  that wasn't completly parsed and the updated state of the NALu parser. If the
-  `should_skip_last_nalu?` option is set to `false`, the last NALu in the list might
-  not be complete, as it might be produced only out of a part of that NALu's binary.
+  that wasn't completly parsed and the updated state of the NALu parser.
```

Comment:
We no longer return payload leftover

Reply:
indeed, I have generally rewritten docs in that module

Reply:
done